src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Templates/TemplateInserterStep.cs

```diff
@@ -259,7 +259,7 @@ private bool ItemMatches(string filePath, ItemSpec expectedItem)
             if (expectedItem.Keywords.Any())
             {
                 var fileContents = File.ReadAllText(filePath);
-                if (expectedItem.Keywords.Any(k => !fileContents.Contains(k, StringComparison.Ordinal)))
+                if (expectedItem.Keywords.Any(k => !fileContents.Contains(k)))
```

Comment:
is this intended as a case-sensitive search?

Reply:
Yes, it's intended to be case-sensitive.